concepts/Common/Licensing/30 FAQ.md

```diff
@@ -40,7 +40,7 @@ A:
 
 - [DevExtreme jQuery](https://js.devexpress.com/jQuery/)
 
-And all related supplemental products like Application Templates, UI Template Gallery, and Demos (if you download them or copy to CodePen or CodeSandBox). At present, license verification is not required for ASP.NET Core/MVC and DevExtreme Reactive.
+And all related supplemental products like Application Templates, UI Template Gallery, and Demos (if you download them or copy to CodePen or CodeSandBox). If you use trial version of DevExtreme in ASP.NET Core/MVC projects, you will see an on-screen message. At present, license verification is not required for DevExtreme Reactive.
```

Comment:
Let's provide ASP users with the link to the licensing topic on devexpress.com https://docs.devexpress.com/GeneralInformation/403579/trial-register/convert-a-trial-version-to-a-licensed-version?v=24.1 
Because they have different way to perform license validation